classes/admin.php

```diff
@@ -745,7 +745,11 @@ public function getPage($path)
                 // Found the type and header from the session.
                 $data = $this->session->{$page->route()};
 
-                $header = ['title' => $data['title']];
+                // Initialise the page headers with the default page headers
+                $default_headers = $this->grav['config']->get('plugins.admin.editor.default_headers');
+                $header = Yaml::parse($default_headers);
+
+                $header['title'] = $data['title'];
```

Comment:
Yeah good point. I've changed it so the default headers are merged in after the title is set

I've also made a slight change to how the `body_classes = modular` is set, so it's concatenated with whatever's set in `default_headers`